packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -23,19 +45,56 @@ const useContentStyles = makeStyles({
   selected: typographyStyles.body1Strong,
 });
 
+const useIndicatorStyles = makeStyles({
+  base: {
+    '::after': {
+      position: 'absolute',
+      transform: `translateX(${navItemTokens.indicatorOffset})`, // per spec
+      backgroundColor: tokens.colorNeutralForeground2BrandSelected,
+      width: navItemTokens.indicatorWidth,
+      height: navItemTokens.indicatorHeight,
+      ...shorthands.borderRadius(tokens.borderRadiusCircular),
+      content: '""',
+    },
+  },
+});
+
+const useIconStyles = makeStyles({
+  base: {
+    height: navItemTokens.indicatorHeight,
+    [`& .${iconClassNames.filled}`]: {
+      display: 'none',
+    },
+    [`& .${iconClassNames.regular}`]: {
+      display: 'inline',
+    },
```

Comment:
based on previous comment
```suggestion
    [`& .${iconFilledClassName}`]: {
      display: 'none',
    },
    [`& .${iconRegularClassName}`]: {
      display: 'inline',
    },
```

Reply:
updated :-)